packages/e2e-test-utils/src/is-in-default-block.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Checks if the block that is focused is the default block.
+ *
+ * @return {Promise} Promise resolving with a boolean indicating if the focused block is the default block.
+ */
+export function isInDefaultBlock() {
+	return page.evaluate( () => {
+		const activeBlockName = document.activeElement
+			.closest( '[data-type]' )
```

Comment:
`document.activeElement` can be `null`:

https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/activeElement

At which point this would throw:

>Uncaught TypeError: Cannot read property 'closest' of null

I'm not sure how well `evaluate` will tolerate the thrown error, but it seems reasonable that we'd just `return false;` if there is no active element.

Reply:
Hi @aduth, nice catch the verification was added.